client/src/gotchi/evolver.ts

```diff
@@ -67,7 +67,7 @@ export class Evolver {
     }
 
     public mutateGenome(mutationCount: number): void {
-        console.log(`mutating ${this.index} ${Direction[this.nextDirection]} ${mutationCount} dice`)
+        // console.log(`mutating ${this.index} ${Direction[this.nextDirection]} ${mutationCount} dice`)
```

Comment:
Remove

Reply:
I'd rather keep it, so i can activate while debugging.

Reply:
Mkay